tests/pagai/conftest.py

```diff
@@ -44,20 +44,10 @@ def table_exists(sql_engine, table_name):
         return False, None
 
 
-@pytest.fixture(scope="session", params=list(DATABASES.keys()))
+@pytest.fixture(params=list(DATABASES.keys()))
 def db_config(request):
     db_driver = request.param
-    db_config = DATABASES[db_driver]
-
-    sql_engine = create_engine(get_sql_url(db_driver, db_config))
-
-    # Load (or reload) test data into db.
-    load_table(sql_engine, "patients", "patients.csv")
-    load_table(sql_engine, "UPPERCASE", "patients-uppercase.csv")
-    load_table(sql_engine, "CaseSensitive", "patients-case-sensitive.csv")
-
-    db_config["model"] = db_driver
-    return db_config
+    return {**DATABASES[db_driver], "model": db_driver}
```

Comment:
nit: could we slightly clean this db_config stuff by having the model key directly in `DATABASES`?

Reply:
yes done